### DISCUSSION TITLE
PCL2 的代码公开度是又增加了吗

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#4017] [提问] [answered]
好一段时间没有来看仓库了，发现 ModSecret.vb 出现在代码里了
然后翻了翻 .gitignore，也没有隐藏了什么别的文件了
想问一下现在的项目不能不能直接编译调试运行了（

**Comment by WForst-Breeze** [comment, marked as answer]
可以了，且已经有一些衍生版本出现了

见#3910
